app/routes/_auth.login.tsx

```diff
@@ -47,6 +47,8 @@ export async function loader({ request }: LoaderArgs) {
 
 export default function Component() {
   const actionData = useActionData();
+  const navigation = useNavigation();
+  const isPending = navigation.state === 'submitting' || navigation.state === 'loading';
```

Comment:
Using Remix's `useNavigation` global navigation object to infer if we should display a pending UI.